client/src/pages/DisplayData.css

```diff
@@ -0,0 +1,23 @@
+:root {
+  --header-colour: #37b0f0;
+  --white-colour: #ffffff;
```

Comment:
We should try to use colors of the Theme in chakra so it's consistent everywhere (also configured for day/night mode).

To setup a color for day/night mode we can call a function `useColorModeValue(lightModeValue, darkModeValue)` from chakra.

We can also use colors from chakra, such as `teal.100`, and `teal.900` from light to dark.

Reply:
Done